doc/languages-frameworks/beam.xml

```diff
@@ -181,6 +240,27 @@ $ nix-env -f &quot;&lt;nixpkgs&gt;&quot; -iA beamPackages.ibrowse
          };
        }
       </programlisting>
+      <para>
+        Alternatively one can use <literal>buildHex</literal> as a shortcut for the above:
+      </para>
+      <programlisting>
+        { buildHex, buildMix, plug, absinthe }:
+        buildHex {
+          name = "absinthe_plug";
+          version = "1.0.0";
+          sha256 =
+              "08459823fe1fd4f0325a8bf0c937a4520583a5a26d73b193040ab30a1dfc0b33";
+          builder = buildMix;
+          beamDeps  = [ plug absinthe];
```

Comment:
This is just stylistic, but maybe put a space between "absinthe" and the closing "]".